target/linux/ar71xx/files/arch/mips/ath79/mach-gs-minibox-v32.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Atheros GS_MINIBOX_V3.2 reference board support
+ *
+ * Copyright (c) 2013 The Linux Foundation. All rights reserved.
+ * Copyright (c) 2012 Gabor Juhos <juhosg@openwrt.org>
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *
+ */
+
+#include <linux/pci.h>
+#include <linux/platform_device.h>
+#include <linux/ath9k_platform.h>
+#include <linux/ar8216_platform.h>
+
+#include <asm/mach-ath79/ar71xx_regs.h>
+
+#include "common.h"
+#include "dev-eth.h"
+#include "dev-ap9x-pci.h"
+#include "dev-gpio-buttons.h"
+#include "dev-leds-gpio.h"
+#include "dev-m25p80.h"
+#include "dev-spi.h"
+#include "dev-usb.h"
+#include "dev-wmac.h"
+#include "machtypes.h"
+#include "pci.h"
+
+#define GS_MINIBOX_V3_GPIO_LED_STATUS		14
+#define GS_MINIBOX_V3_GPIO_BTN_RST                 17
+#define GS_MINIBOX_V3_KEYS_POLL_INTERVAL           20	/* msecs */
+#define GS_MINIBOX_V3_KEYS_DEBOUNCE_INTERVAL	(3 * GS_MINIBOX_V3_KEYS_POLL_INTERVAL)
+#define GS_MINIBOX_V3_MAC0_OFFSET                  0
+#define GS_MINIBOX_V3_MAC1_OFFSET                  6
+#define GS_MINIBOX_V3_WMAC_CALDATA_OFFSET          0x1000
```

Comment:
all the defines above should use tabs and not spaces as separators

Reply:
Sorry for messing tabs and spaces, I'll resend patch